scripts/error_codes.py

```diff
@@ -227,7 +227,8 @@ def examine_id_coverage(top_dir, source_id_to_file_names, new_ids_only=False):
         "1988", "2066", "3356",
         "3893", "3996", "4010", "4802",
         "5272", "5622", "7128", "7400",
-        "7589", "7593", "8065", "8084", "8140",
+        "7589", "7593", "7649", "7710",
+        "8065", "8084", "8140",
```

Comment:
Will remove

Reply:
@cameel actually the error script does not take command line tests into account, that's why we need to blacklist those two errors here

Reply:
I think it must be buggy in that case, because the code for doing that is there :)

https://github.com/ethereum/solidity/blob/f97fe813ec14edb8580061f0082f30b73affc04d/scripts/error_codes.py#L187-L190

But if everything is fine on your side then I guess it has nothing to do with your PR and we can just report it as a separate issue.

Reply:
Ah ok. Yea that's weird, I did remove them from the list after your comment, then the test started failing again.